src/kili/mutations/data_connection/queries.py

```diff
@@ -0,0 +1,11 @@
+"""
+Queries of data connection mutations
+"""
+
+GQL_ADD_PROJECT_DATA_CONNECTION = """
+mutation ($data: DataConnectionInput!) {
```

Comment:
```suggestion
mutation addDataConnection($data: DataConnectionInput!) {
```

So in apollo studio, we can see the name of the call `addDataConnection` instead of seeing the stringified query. Much easier to read